ca/certificates.go

```diff
@@ -815,6 +817,11 @@ func CreateRootCA(rootCN string) (RootCA, error) {
 		return RootCA{}, err
 	}
 
+	key, err = pkcs8.ConvertECPrivateKeyPEM(key)
```

Comment:
By converting the key to pkcs8, won't this CA key only be usable by nodes running a recent enough version?

It seems better to me to only do this when `GOFIPS` is set.